Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -369,59 +371,64 @@ final class IfconfigExprTests: XCTestCase {
       #endif
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected value in function call", fixIts: ["insert value"])
-      ]
+        DiagnosticSpec(message: "expected version tuple in 'canImport' expression in if config expression", fixIts: ["insert version tuple"])
+      ],
+      fixedSource: """
+        #if canImport(A, _version: <#integer literal#>)
+          let a = 1
+        #endif
+        """
     )
   }
 
   func testIfconfigExpr27() {
     assertParse(
       #"""
-      #if canImport(A, _version: "")
+      #if canImport(A, _version: 1️⃣"")
         let a = 1
       #endif
       """#,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: _version argument cannot be empty
+        DiagnosticSpec(message: "cannot parse version code '\"\"'")
```

Comment:
Where is the `code` coming from here?

Reply:
I think the `code` here is because I used `shortSingleLineContentDescription` for the node, already removed it.